Modules/HMPID/src/HmpidTask.cxx

```diff
@@ -125,16 +158,16 @@ void HmpidTask::monitorData(o2::framework::ProcessingContext& ctx)
       if (!mDecoder->decodeBufferFast()) {
         ILOG(Error) << "Error decoding the Superpage !" << ENDM;
```

Comment:
If the task starts receiving bad data it will spam the infologger. Please add at least a level (see suggestion) and if possible use the auto-mute described ([here](https://github.com/AliceO2Group/InfoLogger/blob/master/doc/README.md)).
```suggestion
        ILOG(Error, Devel) << "Error decoding the Superpage !" << ENDM;
```